core/src/jsMain/kotlin/ScannerBuilder.kt

```diff
@@ -1,20 +1,21 @@
 package com.juul.kable
 
-import com.benasher44.uuid.Uuid
 import com.juul.kable.logs.Logging
 import com.juul.kable.logs.LoggingBuilder
 
 public actual class ScannerBuilder {
-    public actual var services: List<Uuid>? = null
+    public actual var filters: List<Filter>? = null
     private var logging: Logging = Logging()
 
     public actual fun logging(init: LoggingBuilder) {
         logging = Logging().apply(init)
     }
 
     internal actual fun build(): Scanner {
-        val filters = services
+        val filters = filters
             ?.map { it.toString() }
```

Comment:
Need to drop the string conversion here now that we have `Filter` objects and the next chain operator will do a filter on the type (and we don't want it to already all be strings).

```suggestion
```

Reply:
Oh, good catch! That would've been an annoying silent failure.